test-apps/display-test-app/package.json

```diff
@@ -31,17 +31,13 @@
     "test": "",
     "cover": "",
     "copy:config": "internal-tools copy-config",
-    "build:ios-backend": "npm run -s build:backend && webpack --config ./mobile_backend.config --env outdir=./lib/ios --env entry=./lib/backend/MobileMain.js --env bundlename=main",
-    "copy:ios-frontend": "mkdir -p ./lib/ios && rm -rf ./lib/ios/www && cp -R ./build ./lib/ios/www",
-    "build:ios": "npm run -s build:frontend-webpack && npm run -s build:ios-backend && npm run -s copy:ios-frontend && npm run build:ios-env-json",
-    "build:ios-env-json": "ts-node --skip-project ./buildEnvJson.ts",
+    "build:mobile": "npm run -s build:mobile-frontend && npm run -s build:mobile-backend && npm run -s build:mobile-env-json",
+    "build:mobile-frontend": "PUBLIC_URL=./ npm run -s build:frontend-webpack",
+    "build:mobile-backend": "npm run -s build:backend && webpack --config ./mobile_backend.config --env outdir=./lib/mobile --env entry=./lib/backend/MobileMain.js --env bundlename=main",
+    "build:mobile-env-json": "ts-node --skip-project ./buildEnvJson.ts",
```

Comment:
nit: dont understand why we need a ts file for this, and use ts-node with the skipProject flag; might as well just be a js script at that point

Reply:
buildEnvJson.ts isn't new for this PR, I've just changed the npm script name to mobile from ios.

Is using ts-node undesirable? We have another ts script (runIosSimulator.ts).

Reply:
yea i noticed this wasnt new so added the nit; up to you ig you want to address this comment, in this pr or another or at all

nbd, but i didnt see a big value add for your script where the only types youre using are strings

Reply:
We could definitely consider changing both scripts to js in another PR. I didn't write buildEnvJson.ts so I hadn't scrutinized it closely.